README.md

```diff
@@ -184,6 +184,50 @@ Hit Ctrl-C to quit.
 </details>
 
 
+## OptunaHub
+
+[OptunaHub](https://hub.optuna.org/) is a feature-sharing platform for Optuna.
+You can use the registered features and publish your packages.
+
+![OptunaHub](https://github.com/user-attachments/assets/cc48e8e9-7307-47e7-8083-4dc6a695995e)
```

Comment:
The file size of this image is about 4MB, and I guess it might degrade the performance score of the website.  Could you remove it from this PR, please?

Reply:
Images are effective in attracting attention.
Would you be able to use a smaller gif?
For example, a 640 × 400 gif is in [the web-dashboard section](https://github.com/optuna/optuna?tab=readme-ov-file#web-dashboard).

Reply:
> Images are effective in attracting attention.

Agree.
But how about adding it in a follow up PR?

Reply:
I removed the gif file. PTAL.